docs/get-started/svelte-vite.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Storybook for Svelte & Vite
+---
+
+export const SUPPORTED_RENDERER = 'svelte';
+
+Storybook for Svelte & Vite is a [framework](../contribute/framework.md) that makes it easy to develop and test UI components in isolation for applications using [Svelte](https://svelte.dev/) built with [Vite](https://vitejs.dev/).
+
+<If notRenderer={SUPPORTED_RENDERER}>
+
+<Callout variant="info">
+
+Storybook for Svelte & Vite is only supported in [Svelte](?renderer=svelte) projects.
+
+</Callout>
```

Comment:
Isn't this kinda obvious? I'm not sure what the user is gaining from reading this.

(I missed this in the Svelte-webpack PR but see that it's there too, and my comment would be the same.)

Reply:
Oohh I see this is in a "you're not in the Svelte docs mode" condition. then it makes total sense.

Reply:
@JReinhold — Yeah, because the snippets are _only_ available for the relevant renderer, I had to wrap all of the content (minus the intro, for some context) in the `If` block to prevent the page erroring when being built or when viewing with a different renderer selected.